CiarnuroRT/webapp/style.css

```diff
@@ -24,12 +24,12 @@ html,body{
     overflow:hidden;
     background:#000000;
     color:#FFFFFF;
-    text-shadow:0.05em 0.05em 0.125em #000000;
+    text-shadow:0.05rem 0.05rem 0.125rem #000000;
     font-family:'Open Sans',sans-serif;
 }
 #loadOverlay{
     position:fixed;
-    top:0; left:0; width:100vw; height:100vh;
```

Comment:
vw is a modern unit and should be used instead of % in these cases, since loadOverlay is meant to cover the entire screen.